joomla/joomla-cms
# [4.4] Update deleted files list in script.php for 4.4.0-alpha1

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the list of files to be deleted on update in file "administrator/components/com_admin/script.php" to recent changes in the 4.4-dev branch in preparation for the first 4.4.0-alpha1 release.

In detail deleted files from following PR's are added:

- PR #40066 file
'/modules/mod_footer/mod_footer.php',
- PR #40092 file
'/modules/mod_users_latest/mod_users_latest.php',
- PR #40142 file
'/plugins/content/pagenavigation/pagenavigation.php',
- PR #40146 files
'/plugins/installer/folderinstaller/folderinstaller.php',
'/plugins/installer/override/override.php',
'/plugins/installer/packageinstaller/packageinstaller.php',
'/plugins/installer/urlinstaller/urlinstaller.php',
'/plugins/installer/webinstaller/webinstaller.php',
- PR #40174 files
'/plugins/privacy/actionlogs/actionlogs.php',
'/plugins/privacy/consents/consents.php',
'/plugins/privacy/contact/contact.php',
'/plugins/privacy/content/content.php',
'/plugins/privacy/message/message.php',
'/plugins/privacy/user/user.php',
- PR #40180 files
'/plugins/user/contactcreator/contactcreator.php',
'/plugins/user/joomla/joomla.php',
'/plugins/user/profile/profile.php',
'/plugins/user/terms/terms.php',
'/plugins/user/token/token.php',
- PR #39527 file
'/modules/mod_articles_category/mod_articles_category.php',
- PR #39535 file
'/modules/mod_articles_archive/mod_articles_archive.php',
- PR #40169 files
'/plugins/media-action/crop/crop.php',
'/plugins/media-action/resize/resize.php',
'/plugins/media-action/rotate/rotate.php',
- PR #40173 files
'/plugins/sampledata/blog/blog.php',
'/plugins/sampledata/multilang/multilang.php',
- PR #39552 file
'/modules/mod_articles_categories/mod_articles_categories.php',
- PR #39931 file
'/modules/mod_custom/mod_custom.php',
- PR #39628 file
'/modules/mod_related_items/mod_related_items.php',
- PR #40328 file
'/modules/mod_breadcrumbs/mod_breadcrumbs.php',
- PR #40216 files
'/plugins/system/accessibility/accessibility.php',
'/plugins/system/actionlogs/actionlogs.php',
'/plugins/system/debug/debug.php',
'/plugins/system/fields/fields.php',
'/plugins/system/highlight/highlight.php',
'/plugins/system/httpheaders/httpheaders.php',
'/plugins/system/jooa11y/jooa11y.php',
'/plugins/system/languagecode/languagecode.php',
'/plugins/system/languagefilter/languagefilter.php',
'/plugins/system/log/log.php',
'/plugins/system/logout/logout.php',
'/plugins/system/logrotation/logrotation.php',
'/plugins/system/privacyconsent/privacyconsent.php',
'/plugins/system/redirect/redirect.php',
'/plugins/system/remember/remember.php',
'/plugins/system/schedulerunner/schedulerunner.php',
'/plugins/system/sef/sef.php',
'/plugins/system/sessiongc/sessiongc.php',
'/plugins/system/skipto/skipto.php',
'/plugins/system/stats/stats.php',
'/plugins/system/tasknotification/tasknotification.php',
'/plugins/system/updatenotification/updatenotification.php',
- PR #40561 files
'/plugins/content/finder/finder.php',
'/plugins/content/joomla/joomla.php',
'/plugins/content/loadmodule/loadmodule.php',
'/plugins/content/pagebreak/pagebreak.php',
'/plugins/content/vote/vote.php',
- PR #40190 files
'/plugins/webservices/banners/banners.php',
'/plugins/webservices/config/config.php',
'/plugins/webservices/contact/contact.php',
'/plugins/webservices/content/content.php',
'/plugins/webservices/installer/installer.php',
'/plugins/webservices/languages/languages.php',
'/plugins/webservices/media/media.php',
'/plugins/webservices/menus/menus.php',
'/plugins/webservices/messages/messages.php',
'/plugins/webservices/modules/modules.php',
'/plugins/webservices/newsfeeds/newsfeeds.php',
'/plugins/webservices/plugins/plugins.php',
'/plugins/webservices/privacy/privacy.php',
'/plugins/webservices/redirect/redirect.php',
'/plugins/webservices/tags/tags.php',
'/plugins/webservices/templates/templates.php',
'/plugins/webservices/users/users.php',
- PR #40205 files
'/plugins/workflow/featuring/featuring.php',
'/plugins/workflow/notification/notification.php',
'/plugins/workflow/publishing/publishing.php',

### Testing Instructions

Code review.

Or if you want to make a real test, update a 4.3.0-rc1 or recent 4.3 nightly build to the last 4.4 nightly build to get the actual result, and update a 4.3.0-rc1 or recent 4.3 nightly build to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

The files mentioned above are still present after updating from a 4.3.0-rc1 or recent 4.3 nightly build.

### Expected result AFTER applying this Pull Request

The files mentioned above have been deleted after updating from a 4.3.0-rc1 or recent 4.3 nightly build.

### Link to documentations
Please select:
